generative_data_prep/processors/article_tokenizer.py

```diff
@@ -230,7 +234,7 @@ def process_jsonl(self,
 
             completion, prompt = self._add_space_separator(completion, prompt)
             new_token_ids, new_token_type_ids = self.tokenize(
-                completion, prompt)
+                completion, prompt, prompter_keyword=self.prompter_keyword,  assistant_keyword=self.assistant_keyword, )
```

Comment:
NIT: extra comma at the end? Not sure how this passed syntax

Reply:
NIT do you even have to pass in these arguments? They are already available under self.prompter_keyword in the tokenize function? I think you can just delete this change